src/blockstore/multi_carv2_disk.rs

```diff
@@ -37,18 +37,18 @@ impl MultiCarV2DiskBlockStore {
 
             let mut deltas = Vec::new();
             // For each child in the directory
-            for dir_entry in fs::read_dir(dir)? {
+            for dir_entry in fs::read_dir(dir)?.flatten() {
```

Comment:
I'm not confident flatten() is the right move here but its not terribly important and I want to get these comments out.